src/main/resources/avro/methods.avdl

```diff
@@ -2,6 +2,35 @@
 
 protocol RPC {
 
+/** A query range suitable for inclusion in a positional range request. */
```

Comment:
Do we only want to make queries on `Reference`s? Or do we want to also allow for using this on `Sequence`s that are introduced for new variation not in a `ReferenceSet`?

Why do we have both this and `Region` in `common.avdl`?